constants/limit.go

```diff
@@ -6,6 +6,15 @@ package constants
 
 // Limits and constraints.
 const (
+	// BuildLimitMin defines the minimum value for repo concurrent build limit.
+	BuildLimitMin = 1
+
+	// BuildLimitMax defines the maximum value for repo concurrent build limit.
+	BuildLimitMax = 30
```

Comment:
How do we feel about making this a ratio of the default or allowing admins to set it?

I know for my installation I'm going to want to up the default limit to be 25 which doesn't give a lot of allowances for users to increase on their own. So, I'd propose we either make this max configurable or make it a ratio so it's always got a proportional increase available for users to increase their limit.

Reply:
> How do we feel about making this a ratio of the default or allowing admins to set it?

I'm sure we could make either approach work 😄 

But, if I had to choose I would probably pick allowing admins to configure the max limit (and timeout) for builds.

I think that would be a simpler and more straightforward approach 👍 

> I know for my installation I'm going to want to up the default limit to be 25 which doesn't give a lot of allowances for users to increase on their own

To make sure I'm following properly, sounds like your repos will need to run at least `25` builds concurrently 😃 

If so, do you have any insight into how often you see these repos running `25` or more builds at once?

Like once a day? Few times a day? Hourly?

Trying to get a feel for the activity level of your installation 👍

Reply:
> So, I'd propose we either make this max configurable or make it a ratio so it's always got a proportional increase available for users to increase their limit.

I agree that the server should be updated to override the maximum number of builds with a static number and/or ratio.

Reply:
Doesn't sound like there are any concerns about allowing a configurable max vs ratios?

For now, I'll start working on enabling this functionality in https://github.com/go-vela/server/pull/558

Reply:
> If so, do you have any insight into how often you see these repos running 25 or more builds at once?

> Like once a day? Few times a day? Hourly?

> Trying to get a feel for the activity level of your installation

Hard to say right now because the installation isn't fully up but comparing metrics to another CI in the environment. I imagine I'd have at least 10-15 repos that will be running 50-80 builds an hour and the metrics in the graphs appear bursty. So, sometimes it's a steady chug but others randomly fire large quantities at a time. 

Max configurable sounds great to me!! 😄

Reply:
Thanks for the extra information 😃 

The functionality that was requested has been added to https://github.com/go-vela/server/pull/558